docs/pipelines/release/caching.md

```diff
@@ -454,7 +454,7 @@ pool:
 steps:
   - task: Cache@2
     inputs:
-      key: 'docker | "$(Agent.OS)" | caching-docker.yml'
+      key: 'docker | $(Agent.OS)'
```

Comment:
revert to 'docker | "$(Agent.OS)" | caching-docker.yml'

Reply:
Done :)